.github/workflows/node.js.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
awesome ! you can specify here the working directory, when the push or pr matches the path then the action is triggered, do you want to add the part to match the file path?
I need to add it as well on the other actions